### PR TITLE
DEVELOPER-2533 - resources claims there are no results before display…

### PR DESCRIPTION
**Pull request by Dantheman720**

### CODE REVIEW

**Comment by Dantheman720** [comment]
retest this please

**Comment by Dantheman720** [comment]
retest this please

**Comment by LightGuard** [comment]
@Dantheman720 Is there still work to be done on this?

**Comment by Dantheman720** [comment]
@LightGuard This is ready to be reviewed. I commented on the JIRA ticket and hoped that @wesbos would be able to take a look at it before it was merged. It looks fine to me, but wanted to make sure.

**Comment by LightGuard** [comment]
That looks like it's working the way I would expect it to. @wesbos can you review and I'll merge if you're happy with it.

File: _partials/developer-materials-results.html.slim [hunk 1]
```diff
@@ -84,7 +82,8 @@
           p
             | {{item.fields.sys_description | truncate}}
             | &hellip;
-.large-24
+  .panel(ng-show="!data.materials.length && !data.loading") Sorry there were no results, please modify your search query.
```
Comment (wesbos) [inline _partials/developer-materials-results.html.slim]: If we are negating here, it should just use ng-hide

File: _partials/developer-materials-results.html.slim [hunk 2]
```diff
@@ -84,7 +82,8 @@
...
+.large-24(ng-show="!data.loading")
```
Comment (wesbos) [inline _partials/developer-materials-results.html.slim]: same here

**Comment by Dantheman720** [comment]
Will fix.  Thanks @wesbos and @LightGuard

File: _partials/developer-materials-results.html.slim [hunk 3]
```diff
@@ -19,7 +16,7 @@
     /     i.fa.fa-align-justify
     /     |  List
 
-    table.developer-materials-grid(ng-hide="data.layout === 'grid' || !data.layout || data.loading")
+    table.developer-materials-grid(ng-hide="data.layout === 'grid' || hide!data.layout")
```
Comment (LightGuard) [inline _partials/developer-materials-results.html.slim]: Um, what is `hide` here?

Comment (Dantheman720) [inline _partials/developer-materials-results.html.slim]: Ugh...typo

**Pull request merged**